docs/contributing/build_documentation.rst

```diff
@@ -101,14 +101,16 @@ following commands.
     cd /home/spack/buildtest
     source scripts/spack_container/setup.sh
 
-You will need to volume mount **$BUILDTEST_ROOT** into `/home/spack/buildtest` in-order to get  buildtest code-base accessible inside
+You will need to volume mount **$BUILDTEST_ROOT** into `/home/spack/buildtest` in-order to get buildtest code-base accessible inside
 the container.
 
-Once you setup is complete, please run the python script and it will auto-generate the documentation examples::
+Once you setup is complete, the python script that will auto-generate the documentation examples can be invoked by using the folllowing buildtest command ::
```

Comment:
Once **your** setup... Change `you` --> `your`.

For this statement we can convey same message `the python script that will auto-generate the documentation examples can be invoked by using the folllowing buildtest command` --> `Once your setup is complete, you can auto-generate documentation examples by running the following::`

Reply:
@shahzebsiddiqui Noted!